src/openvas.c

```diff
@@ -382,6 +776,48 @@ stop_single_task_scan (void)
   killpg (pid, SIGUSR1);
 }
 
+/**
+ * @brief Creates the main kb and inserts the scan id
+ *
+ * @param scan_id
+ * @return int 0 on success, -1 on failure
+ */
+int
+create_main_kb (const char *scan_id)
+{
+  kb_t main_kb;
+  printf ("db_address: %s\n", prefs_get ("db_address"));
+  if (kb_new (&main_kb, prefs_get ("db_address") ?: KB_PATH_DEFAULT))
```

Comment:
```suggestion
  if (kb_new (&main_kb, prefs_get ("db_address")))
```
The defaults were already set previously.